test/integration/features/generic-driver.feature

```diff
@@ -0,0 +1,49 @@
+@generic-driver
+Feature: With generic driver Minishift can provision remote unprovisioned VM 
+
+  Scenario: User creates an unprovisioned VM
+    Given setting up environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" with value "y" succeeds
+     When executing "minishift profile set remote" succeeds
+      And executing "minishift start --no-provision" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: Provisioning remote instance
+    Given unset environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" succeeds
+      And setting scenario variable "REMOTE_VM_IP" to the stdout from executing "minishift ip"
+      And executing "minishift profile set minishift" succeeds
+     When executing "minishift start --vm-driver generic --remote-ipaddress $(REMOTE_VM_IP) --remote-ssh-key .minishift/profiles/remote/machines/remote/id_rsa --remote-ssh-user docker" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: Stopping with generic driver
+  Stopping with generic driver puts oc cluster down, remote VM however keeps running.
+     When executing "minishift stop" succeeds
+      And executing "minishift status" succeeds
+     Then stdout should match "OpenShift:\s+Stopped"
+
+  Scenario: VM at remote location is still running
+     When executing "minishift status --profile remote" succeeds
+     Then stdout should match "Minishift:\s+Running"
+      And stdout should match "OpenShift:\s+Stopped"
```

Comment:
Please add a step like
```
And executing “minishift ssh echo test” succeeds
```
This will ensure that VM is running and can also be accessible despite openshift cluster is down

Reply:
This is something we really don't care about when we do `minishift stop` on the remote. This scenario can be removed also.